examples/LPBatching/lpBatching.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"math/rand"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	"github.com/InfluxCommunity/influxdb3-go/influxdb3"
+	"github.com/InfluxCommunity/influxdb3-go/influxdb3/batching"
+	"github.com/apache/arrow/go/v15/arrow"
+)
+
+const LineCount = 100
+
+func main() {
+	// PREPARE DATA
+	// Create a random number generator
+	rnd := rand.New(rand.NewSource(time.Now().UnixNano()))
+
+	// Retrieve credentials from environment variables.
+	url := os.Getenv("INFLUX_URL")
+	token := os.Getenv("INFLUX_TOKEN")
+	database := os.Getenv("INFLUX_DATABASE")
+
+	// unmanned aquatic surface vehicle as source
+	dataTemplate := "uasv,id=%s,location=%s speed=%f,bearing=%f,ticks=%di %d"
+	ids := []string{"orca", "flipper", "gaston"}
+	syncLocations := []string{"nice", "split", "goa", "cancun"}
+
+	// Instantiate a client using your credentials.
+	client, cErr := influxdb3.New(influxdb3.ClientConfig{
```

Comment:
Why different errors, cErr, wErr? A single `err` should be sufficient.

Reply:
I thought it useful to distinguish between errors that arise on Write and errors that arise on client creation. however in the end this is not really needed.  Updated.